Xamarin.Forms.Platform.Android/Renderers/ListViewAdapter.cs

```diff
@@ -535,6 +505,53 @@ void SelectItem(object item)
 			Select(position, view);
 		}
 
+		void UpdateSeparatorVisibility(Cell cell, bool cellIsBeingReused, bool isHeader, bool nextCellIsHeader, ConditionalFocusLayout layout, out AView bline)
+		{
+			bline = null;
+			if (cellIsBeingReused)
```

Comment:
Having this in here kind of complicates the flow unnecessarily, I think. I'd move lines 510-512 out of here and only call `UpdateSeparatorVisibility` and `UpdateSeparatorColor` if the cell is not being reused.